README.md

```diff
@@ -1,18 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome Books Project 
 
-> One paragraph statement about the project.
 
-![screenshot](./app_screenshot.png)
+> This project builds on a basic website that allows users to add/remove books from a list. It is achieved by using JavaScript classes with the help of dynamically modifying the DOM and adding basic events.
+
+![Screenshot (70)](https://user-images.githubusercontent.com/61976324/148301182-f7a12595-f658-4b13-9244-7c36127457ea.png)
+
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML, CSS and Javascript.
+- Frameworks: No
+- Technologies used: VScode, chrome, and github
 
 ## Live Demo
 
```

Comment:
- [ ] For all the subheadings in your `readme` file where there is no information, it will be nice to delete them, as this will make your `readme` cleaner and professional.

Reply:
You can see I have many linter problem,  I've forgot readme totally.  Thanks but I think that doesn't allow you to approve this project